### PR TITLE
media-libs/libde265: bump to 1.0.8

### PR DESCRIPTION
**Pull request by jsmolic**

### CODE REVIEW

File: media-libs/libde265/libde265-1.0.8.ebuild [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools multilib-minimal
+
+PATCHES=( "${FILESDIR}/${PN}-1.0.2-qtbindir.patch" )
```
Comment (SoapGentoo) [inline media-libs/libde265/libde265-1.0.8.ebuild:8]: move down

File: media-libs/libde265/libde265-1.0.8.ebuild [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+if [[ ${PV} == "9999" ]] ; then
```
Comment (SoapGentoo) [inline media-libs/libde265/libde265-1.0.8.ebuild:10]: you know what to do

File: media-libs/libde265/libde265-1.0.8.ebuild [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+#		media-libs/libsdl
+#		dev-qt/qtcore:5
+#		dev-qt/qtgui:5
+#		dev-qt/qtwidgets:5
+#		media-libs/libswscale
+#	)
+#"
+
+DEPEND="${RDEPEND}"
+
```
Comment (SoapGentoo) [inline media-libs/libde265/libde265-1.0.8.ebuild:43]: remove empty line

File: media-libs/libde265/libde265-1.0.8.ebuild [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+	# without this, headers would be missing and make would fail
+	multilib_copy_sources
+}
+
+multilib_src_configure() {
+	local myeconfargs=(
+		ax_cv_check_cflags___msse4_1=$(usex cpu_flags_x86_sse4_1)
+		ax_cv_check_cflags___mfpu_neon=$(usex cpu_flags_arm_neon)
+		$(use_enable cpu_flags_arm_thumb thumb)
+		$(use_enable static-libs static)
```
Comment (SoapGentoo) [inline media-libs/libde265/libde265-1.0.8.ebuild:63]: :grinning:

File: media-libs/libde265/libde265-1.0.8.ebuild [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+		# Remove all non-native binary tools
+		rm -f "${ED}/usr/bin/"* || die
+	fi
+}
+
+multilib_src_install_all() {
+	find "${ED}" -name '*.la' -delete || die
+	if ! use static-libs ; then
+		find "${ED}" -name "*.a" -delete || die
+	fi
```
Comment (SoapGentoo) [inline media-libs/libde265/libde265-1.0.8.ebuild:98]: simplify and what about `einstalldocs`?

File: media-libs/libde265/libde265-1.0.8.ebuild [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+	econf "${myeconfargs[@]}"
+}
+
+multilib_src_install() {
+	default
+
+	if multilib_is_native_abi; then
+		# Remove useless, unready and test tools (Why package ever tried to install them??)
+		rm -f "${ED}/usr/bin/"{tests,gen-enc-table,yuv-distortion} || die
```
Comment (SoapGentoo) [inline media-libs/libde265/libde265-1.0.8.ebuild:83]: you know, quotes only around `${ED}` etc

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: media-libs/libde265/libde265-1.0.8.ebuild (hunk 1 above)
Comment (jsmolic) [reply media-libs/libde265/libde265-1.0.8.ebuild]: Done

File: media-libs/libde265/libde265-1.0.8.ebuild (hunk 2 above)
Comment (jsmolic) [reply media-libs/libde265/libde265-1.0.8.ebuild]: :smile:

File: media-libs/libde265/libde265-1.0.8.ebuild (hunk 6 above)
Comment (jsmolic) [reply media-libs/libde265/libde265-1.0.8.ebuild]: Done

File: media-libs/libde265/libde265-1.0.8.ebuild [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+SLOT="0"
+IUSE="enc265 dec265 sdl tools debug cpu_flags_x86_sse4_1 cpu_flags_arm_neon cpu_flags_arm_thumb"
+# IUSE+=" sherlock265" # Require libvideogfx or libswscale
+
+RDEPEND="
+	dec265? (
+		sdl? ( media-libs/libsdl )
+	)"
+
+# Sherlock265 require libvideogfx or libswscale
```
Comment (jsmolic) [inline media-libs/libde265/libde265-1.0.8.ebuild:29]: I can leave this stuff commented in case anyone feels inspired to package new deps

**Pull request closed without merging**